libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/history/ReplayHistoryDTO.kt

```diff
@@ -2,6 +2,14 @@ package com.mapbox.navigation.core.replay.history
 
 import com.google.gson.annotations.SerializedName
 
+/**
+ * DTO of ReplayHistory. If you'd like to add a custom event, add it to [events], and
+ * pass a [CustomEventMapper] to the [ReplayHistoryMapper].
+ *
+ * @param historyVersion version of history
+ * @param version // todo add docs
+ * @param events include custom
```

Comment:
```suggestion
 * Data representation of the string given by [MapboxNavigation.retrieveHistory]. If you'd like to 
 * add a custom event, add it to [events] and pass a [CustomEventMapper] to the [ReplayHistoryMapper].
 *
 * @param historyVersion version of the events supported: 1.0.0
 * @param version version of the navigator library
 * @param events navigation events to replay
```

 * Data representation of the string given by [MapboxNavigation.retrieveHistory]. If you'd like to 
 * add a custom event, add it to [events] and pass a [CustomEventMapper] to the [ReplayHistoryMapper].